src/TimeoutMigrationTool.Raven4.Core8.AcceptanceTests/ConfigureExtensions.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace TimeoutMigrationTool.Raven4.Core8.AcceptanceTests
```

Comment:
does this intentionally use a different namespace?

Reply:
It's not :)